### ISSUE TITLE
pipeline operator + arrow function breaks prettier

### ISSUE THREAD

**Issue by avetisk** [prettier/prettier#7669]
**Prettier 1.19.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOEA2cB0mIDmAFAIwBMAzAAQA+AfFQB5UC8DzA1FSQJQgA0ICAAcYAS3TJQAQwBOsiAHcACnIRpkIaQDcIYgCYCQAI1nSwAazgwAysPNioBZDFkBXOIIAWMALaYAdS8xeDR7MDgbdRCxbRCAT00wNA1BRzQ4WRhlMwJfaWQAM2lMDMEAKzRGACEzS2sbaV84ABlHOCKSspBKxhtHAmwARTcIeE7SzxB7WQzZTWNpYzhMI2FZRxgAgxgvZAAOAAZBdYgMgLNhTXW4Oe0OwQBHUfgckQ0UaTQAWig4OH0AKMsjgzzEIJy0jyBSQxUmggyvjELncUzQA2GLw6sK6UxgS22+l2yDIglc0jEmAGAGEIL58pooNAHiA3BkACpLD5w7raDwASSggNgNjAG1EAEEhTYYPFsBMMgBfRVAA)

Prettier config:
```js
{
  useTabs: false,
  printWidth: 80,
  tabWidth: 2,
  singleQuote: true,
  trailingComma: 'es5',
  bracketSpacing: true,
  jsxBracketSameLine: false,
  semi: false,
}
```

Babel plugin for pipeline operator:
```js
['@babel/plugin-proposal-pipeline-operator', { proposal: 'fsharp' }]
```

**Input:**
```js
console.log(123 |> x=>x+1)
```

**Output:**
```js
console.log(123 |> x=>x+1)
```

**Expected behavior:**
```js
console.log(123 |> x => x + 1)
```

So basically as long as I don't put arrow functions with a pipeline operator, everything's fine.

But if I use them together, prettier just doesn't do anything, it leaves the code as it is.

**Comment by thorn0** [comment]
Prettier doesn't support smart and f# pipelines yet.

**Comment by thorn0** [comment]
Duplicate of #6162
